pyproject.toml

```diff
@@ -34,7 +34,7 @@ dependencies = [
 "sentencepiece>=0.1.99,<0.3",
 "tokenizers>=0.13.3,<1.0",
 "tqdm>=4.66.2,<5.0",
-"trl>=0.13,<0.18",
+"trl>=0.13,<0.17",
```

Comment:
If we are anyway returning the right collator, should we still revert the version?